### PR TITLE
Release/v0.37.7

### PR DESCRIPTION
**Pull request by greg-szabo**
RELEASE v0.37.7

[CHANGELOG](https://github.com/cometbft/cometbft/blob/1cb3eea5416f8021a792b43c7253794eba0276bb/CHANGELOG.md)
---

#### PR checklist

- [ ] Tests written/updated
- [ ] Changelog entry added in `.changelog` (we use [unclog](https://github.com/informalsystems/unclog) to manage our changelog)
- [ ] Updated relevant documentation (`docs/` or `spec/`) and code comments
- [ ] Title follows the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec

### CODE REVIEW

**Review by hvanz — APPROVED** [review]
👍

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
 # CHANGELOG
 
+## v0.37.7
+
+*June 27, 2024*
+
+This release contains a few bug fixes and performance improvements.
```
Comment (melekes) [inline CHANGELOG.md:7]: I would write something like: `This release is centered around two topics: mempool and performance improvements. The mempool will stop accepting new transactions if the node can't keep up with rechecking the number of transactions already in the mempool. It also contains a few bug fixes.`

**Review by melekes — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (melekes) [reply CHANGELOG.md:7]: basically being a little bit more descriptive

Comment (greg-szabo) [reply CHANGELOG.md]: Yes, thank you! I've added it to the changelog. I felt this too with the 0.38 release. I could've been more descriptive there too.

**Pull request merged**